conf/routes

```diff
@@ -6,12 +6,16 @@
 # An example controller showing a sample home page
 GET     /                           controllers.HomeController.index
 
-GET     /get-generated-schedule                           controllers.ScheduleController.generateScheduleResponse
+GET     /get-generated-schedule                 controllers.ScheduleController.generateScheduleResponse
+GET     /insert-student                         controllers.InsertController.insertStudent
```

Comment:
Would it not be more correct to have a route which takes the form `/student/insert` instead?

Reply:
Also could we have it as a post so I can send the student data in the body